dev-scripts/Dockerfile.agent

```diff
@@ -0,0 +1,91 @@
+ARG RANCHER_TAG=dev
```

Comment:
Let's find a way to not duplicate files, as they will quickly become out of sync - as the Dockerfiles for the integration tests.